x-pack/elastic-agent/pkg/core/plugin/process/app.go

```diff
@@ -117,7 +117,7 @@ func (a *Application) Name() string {
 
 // Started returns true if the application is started.
 func (a *Application) Started() bool {
-	return a.state.Status != state.Stopped
+	return a.state.Status != state.Stopped && a.state.Status != state.Crashed && a.state.Status != state.Failed
```

Comment:
Why this change?

I think `Started()` is trying to say if the application was started at all. If it is crashed or failed you don't want `Start()` to be called again, because that state is handled internally in the application abstraction.

Reply:
yes in case verification fails, it says state to FAILING
when it passes in second round is tries to start the app but according to this check it is already running so it tries to configure it right away. 

i just unified this check with the one in `start.go` so start.go should be unaffected but `operation_start.Check` should start functioning as intended (not skipping operation)